### PR TITLE
`Development`: Improve JPQL style for Hestia repositories

### PR DESCRIPTION
**Pull request by julian-christl**
#### General

- [X] I chose a title conforming to the [naming conventions for pull requests](https://docs.artemis.cit.tum.de/dev/development-process/#naming-conventions-for-github-pull-requests).
#### Server
- [X] I followed the [coding and design guidelines](https://docs.artemis.cit.tum.de/dev/guidelines/server/).

### Motivation and Context

I just fixed the style in one repository so I decided to go through some more repos to make the codebase more consistent and clean.

### Description

I enforced capitalisation (SELECT instead of select), simple variable declaration (:var instead of :#{#var}, explicit enum values, consistent newlines and indentations

#### Code Review
- [x] Code Review 1
- [x] Code Review 2

### Test Coverage

No functional changes.

## Summary by CodeRabbit

- **Refactor**
  - Improved readability and maintainability of JPQL queries in various repositories.

- **Bug Fixes**
  - Updated logic for querying coverage reports to correctly handle submission types.

- **Style**
  - Adjusted query indentation for consistency across the codebase.

- **Documentation**
  - Clarified method signatures and parameter annotations for better understanding.

### CODE REVIEW

**Review by muradium — DISMISSED** [review]
Changes LGTM

**Review by tobias-lippert — APPROVED** [review]
LGTM but the title should be changed from plagiarism to Hestia I guess.

**Review by Hialus — APPROVED** [review]

File: src/main/java/de/tum/in/www1/artemis/repository/hestia/ProgrammingExerciseTaskRepository.java [hunk 1]
```diff
@@ -118,15 +118,15 @@ default Set<ProgrammingExerciseTask> findByExerciseIdWithTestCaseAndSolutionEntr
...
     @Query("""
             SELECT t.taskName
-            FROM ProgrammingExerciseTask t
+                FROM ProgrammingExerciseTask t
             WHERE t.id = :taskId
             """)
     String getTaskName(@Param("taskId") Long taskId);
 
     @Query("""
             SELECT pt FROM ProgrammingExerciseTask  pt
```
Comment (Hialus) [inline src/main/java/de/tum/in/www1/artemis/repository/hestia/ProgrammingExerciseTaskRepository.java:127]: Just noticed: You missed a from here ^^

**Review by Hialus — APPROVED** [review]

**Review by coolchock — APPROVED** [review]
lgtm

**Review by MichaelOwenDyer — APPROVED** [review]

**Review by Hialus — APPROVED** [review]

**Review by tobias-lippert — DISMISSED** [review]

**Review by Strohgelaender — APPROVED** [review]

**Review by coolchock — APPROVED** [review]

**Review by Strohgelaender — APPROVED** [review]

**Pull request merged**